app/services/data/get-individual-overview.js

```diff
@@ -9,7 +9,8 @@ module.exports = function (id) {
     .join('workload_points', 'workload_points_calculations.workload_points_id', 'workload_points.id')
```

Comment:
Should this be in a view.  All other case overview pages are in views and there is alot going on here.  Should more complicated queries be in views?

Reply:
Maybe.....